ilp-core/src/main/java/org/interledger/core/InterledgerAddress.java

```diff
@@ -35,7 +35,7 @@
 public interface InterledgerAddress {
 
   String REGEX = "(?=^.{1,1023}$)"
-      + "^(g|private|example|peer|self|test[1-3])[.]([a-zA-Z0-9_~-]+[.])*([a-zA-Z0-9_~-]+)?$";
+      + "^(g|private|example|peer|self|test([1-3])?)[.]([a-zA-Z0-9_~-]+[.])*([a-zA-Z0-9_~-]+)?$";
 
```

Comment:
👍 - depends on https://github.com/interledger/rfcs/pull/376

Reply:
This change isn't enough to make this work -- we would also need to change the `PREFIX_REGEX`. I made this change (and added unit tests) in a new PR #90. 

I suggest we remove this change from this PR, and use #90 instead.

Reply:
@sappenin  I agree. #90 fixes it properly. 

I don't think I can remove from the PR (or can I?) but it will be immediate to fix during merge-conflict-resolution.

Reply:
Either way is fine - I think you can just remove this change in your branch, and then push another commit (which will just restore this class to its previous state). 

We probably want to do this for the crypto-condition change above as well since these two things have quite a bit of discussion, and are artificially blocking your other changes.